src/transitions/Grow.js

```diff
@@ -0,0 +1,172 @@
+// @flow
+
+import React, { Component } from 'react';
+import type { Element } from 'react';
+import withTheme from '../styles/withTheme';
+import Transition from '../internal/Transition';
+import type { TransitionCallback } from '../internal/Transition';
+
+export function getScale(value: number) {
+  return `scale(${value}, ${value ** 2})`;
+}
+
+type DefaultProps = {
+  theme: Object,
```

Comment:
necessary for flow resolution.  We could add it to `AllProps` instead, but that would differ from the pattern in other files.